### PR TITLE
Interval (gene/transcript/exon) coverage at default threshold

### PR DESCRIPTION
**Pull request by northwestwitch**
### This PR adds | fixes:
- Adds coverage stats at default threshold:

image

### How to test:
- Deployed on stage: https://chanjo2-stage.scilifelab.se/report/demo

### Expected outcome:
- [x] The demo should work

### Review:
- [ ] Code approved by
- [x] Tests executed by CR

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: src/chanjo2/meta/handle_report_contents.py [hunk 1]
```diff
@@ -80,14 +80,48 @@ def get_report_data(query: ReportQuery, session: Session) -> Dict:
...
+def get_report_level_completeness_rows(
+    samples_coverage_stats: Dict[str, List[GeneCoverage]], level: int
+) -> List[Tuple[str, float, str]]:
+    """Create and return the contents of the coverage stats row at the default threshold level."""
+    default_level_rows: List[Tuple[str, float, str]] = []
+
+    for sample, genes_stats in samples_coverage_stats.items():
+        nr_inner_intervals: int = 0
+        covered_inner_intervals: int = 0
+        for gene_stats in genes_stats:  # loop over genes
```
Comment (henrikstranneheim) [inline src/chanjo2/meta/handle_report_contents.py:99]:
```suggestion
        for gene_stats in genes_stats: 
```

Comment (henrikstranneheim) [inline src/chanjo2/meta/handle_report_contents.py:99]: It would be great to make this into a separate unit tested function

**Review by henrikstranneheim — COMMENTED** [review]
More and more complete 👍

File: src/chanjo2/meta/handle_report_contents.py (hunk 1 above)
Comment (northwestwitch) [reply src/chanjo2/meta/handle_report_contents.py]: Good point, testing this and another similar function in that file. Will do!

Comment (northwestwitch) [reply src/chanjo2/meta/handle_report_contents.py]: Ok, tests written. This is ready for a new review when you have time @henrikstranneheim

**Review by henrikstranneheim — APPROVED** [review]
Well done 👍

**Pull request merged**